packages/components/src/experience-control/index.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { ExperienceControl } from './index';
+
+const meta: Meta< typeof ExperienceControl > = {
+	title: 'Components/ExperienceControl',
```

Comment:
This isn't matching the other titles, making this component show up in its own grouping.

```diff
diff --git a/packages/components/src/experience-control/index.stories.tsx b/packages/components/src/experience-control/index.stories.tsx
index 6041c86db05..c2fd511e4c3 100644
--- a/packages/components/src/experience-control/index.stories.tsx
+++ b/packages/components/src/experience-control/index.stories.tsx
@@ -2,7 +2,7 @@ import { Meta, StoryObj } from '@storybook/react';
 import { ExperienceControl } from './index';
 
 const meta: Meta< typeof ExperienceControl > = {
-	title: 'Components/ExperienceControl',
+	title: 'packages/components/ExperienceControl',
 	component: ExperienceControl,
 };
 
```